Build/scripts/pgo/pre_pgo.cmd

```diff
@@ -12,6 +12,13 @@
 :: - build (using PGO profile)
 :: - post_pgo.cmd
 
+@echo off
+
+if "%PogoConfig%"=="False" (
+    echo "---- Not a Pogo Config. Skipping step."

```

Comment:
Assuming %PogoConfig% can only be controlled by the script, that's true, but to insulate in case of other possible values, I'd rather keep the quotes.